.travis.yml

```diff
@@ -2,10 +2,40 @@ language: python
 python:
   - "2.7"
 
+sudo: required
+
 install:
   - pip install pep8
 
+services:
+  - docker
+
 script:
   # Run pep8 on all .py files in all subfolders
   # ignore E402: module level import not at top of file
   - find . -name \*.py -exec pep8 --ignore=E402 {} +
+
+  # Run Ceph Container and Wait for some time so that ceph runs
+  - sudo docker run -d -v ceph:/etc/ceph -e MON_IP=172.17.0.2 -e CEPH_PUBLIC_NETWORK=172.17.0.0/24 --name ceph ceph/demo
```

Comment:
@knikolla as per the build cycle, the pep8 check should go in the script section and running containers should go in before_install section. But it doesnt make sense to run the containers if pep8 failed right ? what do you think ?